ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -120,6 +120,9 @@ Both data models support variable inheritance from profiles defined under [`port
           # Optional if default link_tracking settings are configured on the node.
           name: < tracking_group_name >
 
+        # Structured config added under ethernet_interfaces.<interface>.dot1x for eos_cli_config_gen
```

Comment:
Just note, that based on the way it's done it will require updates if we make modifications to eos_cli_config_gen.
Hopefully, we can fix that with the schema-generated documentation!

Reply:
Correct. With schema we will dynamically fetch the subschema from eos_cli_config_gen.

Reply:
copy/pasted dot1x data model from eos_cli_config_gen README